source/manual/data-gov-uk-2nd-line.html.md

```diff
@@ -55,11 +54,6 @@ There are three environments for CKAN:
 
 [Sentry] monitors application errors. The Sentry pages for each app can be found on the [Find] and [Publish] app pages.
 
-### Log.it
-
-Each application sends logs to [Logit]. [Publish] and [Find] use the corresponding [PaaS Service][logit-paas].
-Example query: `source_host: "gds-data-gov-uk.data-gov-uk.find-data-beta" && access.response_code: 500`.
```

Comment:
Hmmm did this get moved

Reply:
There are a few DGU things that I don't know if they've moved so I removed them for now. I'll ask Ken for a review too